pkg/spec3/benchmark_serialization_test.go

```diff
@@ -0,0 +1,113 @@
+package spec3
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+	"reflect"
+	"testing"
+
+	"k8s.io/kube-openapi/pkg/internal"
+	jsonv2 "k8s.io/kube-openapi/pkg/internal/third_party/go-json-experiment/json"
+)
+
+func TestOpenAPIV3Deserialize(t *testing.T) {
+	swagFile, err := os.Open("./testdata/appsv1spec.json")
```

Comment:
Have you written a fuzzer to complement the comparison here and make sure that we generate the same thing?

Reply:
Yes, have a separate PR for it

Reply:
https://github.com/kubernetes/kube-openapi/pull/370 here's the fuzzer. I've tested this Unmarshal with the fuzzer and can make it a permanent test once some of these PRs are merged since there is quite a bit of a dependency tree with the PRs at the moment.